app/components/views/HomePage/Page.js

```diff
@@ -11,6 +11,46 @@ import TransactionsTab from "./tabs/Transactions";
 import "style/Fonts.less";
 import "style/HomePage.less";
 
+const tabMessages = [
+  <T id="home.tab.balance" m="Balance" />,
+  <T id="home.tab.tickets" m="Tickets" />,
+  <T id="home.tab.transactions" m="Transactions" />,
+];
+
+const tabLink = (i) => {
+  const m = [
+    (
+      <Tooltip text={tabMessages[i]}>
+        <span className="overview-tab-balance">&amp;nbsp;</span>
```

Comment:
Do you really need these `&amp;nbsp;` elements here and in some other spans?

Reply:
removed